justfile

```diff
@@ -134,7 +134,17 @@ run-telemetry: devenv
 
 
 test-all *args: assets
```

Comment:
I agree about renaming this to `test-ci` as it's no longer testing everything. I haven't managed to think of a better name yet!